src/lwc/utilInput/utilInput.html

```diff
@@ -57,6 +57,22 @@
             </c-ge-form-field-picklist>
         </template>
 
+        <template if:true={isTrueFalsePicklist}>
+            <c-ge-form-field-picklist class-name="slds-m-around_none"
```

Comment:
Why is the geFormFieldPicklist component needed here?  Can utilInput use lighting-combobox directly in its template?